scripts/ddetailer.py

```diff
@@ -61,7 +61,13 @@ def startup():
     if (len(list_models(dd_models_path)) == 0):
         print("No detection models found, downloading...")
         load_file_from_url("https://huggingface.co/dustysys/ddetailer/resolve/main/mmdet/bbox/mmdet_anime-face_yolov3.pth", bbox_path)
-        load_file_from_url("https://huggingface.co/dustysys/ddetailer/resolve/main/mmdet/segm/mmdet_dd-person_mask2former.pth", segm_path)
+        if legacy:
+            load_file_from_url("https://huggingface.co/dustysys/ddetailer/resolve/main/mmdet/segm/mmdet_dd-person_mask2former.pth", segm_path)
+        else:
+            load_file_from_url(
+                "https://download.openmmlab.com/mmdetection/v3.0/mask2former/mask2former_r50_8xb2-lsj-50e_coco/mask2former_r50_8xb2-lsj-50e_coco_20220506_191028-41b088b6.pth",
+                segm_path,
+                file_name="mmdet_dd-person_mask2former.pth")
```

Comment:
This is fine for fresh installs but not updates. This would not replace the previous model file when upgrading from mmdet<3. More importantly, nothing seems to be going wrong (no exceptions or warnings are raised) when using the v3 config with the old model other than never detecting anything.

Reply:
good point. it needs to consider cases where the models and config don't match etc.
Simply removing the `models/mmdet` dir will suffice for developers, but for end users more sophisticated method is required such as adding a stamp file and verifying it internally.